src/ResourceManager/Sql/Commands.Sql/Instance Failover Group/Cmdlet/RemoveAzureSqlInstanceFailoverGroup.cs

```diff
@@ -0,0 +1,146 @@
+    {
+        /// <summary>
+        /// Parameter set name for remove with an Input Object.
+        /// </summary>
+        protected const string RemoveIFGByInputObjectParameterSet =
+            "Remove a Instance Failover Group from AzureSqlInstanceFailoverGroupModel instance definition";
+        
+        /// <summary>
+        /// Parameter set name for remove with a resource ID.
+        /// </summary>
+        private const string RemoveIFGByResourceIdSet = "Remove a Instance Failover Group from Resource Id";
+
+        /// <summary>
+        /// Gets or sets the name of the resource group to use.
+        /// </summary>
+        [Parameter(Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            Position = 0,
+            HelpMessage = "The name of the resource group.")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public override string ResourceGroupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the local region to use.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 1,
+            HelpMessage = "The name of the Primary Region of the Instance Failover Group.")]
+        [LocationCompleter]
+        [ValidateNotNullOrEmpty]
+        public string Location { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the InstanceFailoverGroup to remove.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            Position = 2,
+            HelpMessage = "The name of the Azure SQL Database Instance Failover Group to remove.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        /// <summary>
+		/// Instance Failover Group object to remove
+		/// </summary>
+		[Parameter(ParameterSetName = RemoveIFGByInputObjectParameterSet,
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The Instance Failover Group object to remove")]
+        [ValidateNotNullOrEmpty]
+        public Model.AzureSqlInstanceFailoverGroupModel InputObject { get; set; }
+
+        /// <summary>
+        /// Gets or sets the resource ID of the Instance Failover Group to remove.
+        /// </summary>
+        [Parameter(ParameterSetName = RemoveIFGByResourceIdSet,
+            Mandatory = true,
+            Position = 0,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The Resource ID of the Instance Failover Group to remove.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        /// <summary>
+        /// Defines whether it is ok to skip the requesting of rule removal confirmation
+        /// </summary>
+        [Parameter(HelpMessage = "Skip confirmation message for performing the action.")]
+        public SwitchParameter Force { get; set; }
```

Comment:
@jugeorge the `Force` parameter should not be necessary unless you expect to be doing additional prompting to the user

Reply:
I was under the impression that when we do a remove, powershell automatically does an extra "are you sure?" check and this parameter allows the user to surpass that. Is this no longer the case?